example.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import plotly.express as px
+
+import flytekit
+from flytekit import task, workflow
+from flytekit.visualize import FrameProfilingRenderer, FrameRenderer, MarkdownRenderer, ScatterRenderer
+
+
+@task
+def t1() -> str:
+    md_text = "#Hello Flyte\n##Hello Flyte\n###Hello Flyte"
+    m = MarkdownRenderer(md_text)
+    s = ScatterRenderer(range(10), range(10))
+    flytekit.visualize.Deck("Tab1", s).append(m)
+    flytekit.visualize.Deck("Tab2", m)
```

Comment:
is the object usable more than once?  should we have an example where it's saved to a variable and called again?

Reply:
yes, updated it